packages/typespec-test/test/openai/generated/openapi/2022-12-01/openapi.json

```diff
@@ -460,14 +460,9 @@
           "description": "The ordered index associated with this completions choice."
         },
         "logprobs": {
-          "type": "object",
+          "$ref": "#/definitions/CompletionsLogProbabilityModel",
```

Comment:
refer to : https://github.com/Azure/typespec-azure/pull/2576/files#diff-bf98808c51bc72c33ce9f989b62f3d14fbf20e5b91d9f2951aa4fffe90486a27R1837